src/main/kotlin/hwr/oop/projects/peakpoker/core/hand/HandEvaluator.kt

```diff
@@ -1,47 +1,83 @@
 package hwr.oop.projects.peakpoker.core.hand
 
 import hwr.oop.projects.peakpoker.core.card.Card
+import hwr.oop.projects.peakpoker.core.card.HoleCards
+import hwr.oop.projects.peakpoker.core.card.CommunityCards
 import hwr.oop.projects.peakpoker.core.card.Rank
+import hwr.oop.projects.peakpoker.core.hand.HandRank
 
-/**
- * Evaluates a poker hand and returns its ranking.
- */
 object HandEvaluator {
+
     /**
-     * Determines the ranking of the given list of five cards.
-     * @param list of five cards representing the hand
-     * @return the corresponding HandRank
+     * Evaluates the best HandRank from exactly five cards.
      */
-    fun evaluate(cards: List<Card>): HandRank {
-        // Hand must contain exactly five cards and all must be unique
+    internal fun evaluate(cards: List<Card>): HandRank {
```

Comment:
If you want to evaluate the community cards the type of the argument should then match :)
```suggestion
    internal fun evaluate(community: CommunityCards): HandRank {
```